src/db/QueryBuilder.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/** @var \PDO $pdo */
+require_once './pdo_ini.php';
+
+
+class QueryBuilder
+{
+    private $queryParams = [];
+
+    /**
+     * @param $command
+     * @param $param
+     */
+    public function addQueryParam(string $command, string $param)
+    {
+        $this->queryParams[$command] = $param;
+    }
+
+    /**
+     * @return array
+     */
+    public function buildQuery(): array
+    {
+        
+        $params = [];
+        $queryFilter = "";
+        $querySort = "";
+        $queryLimit = "";
+        
+        if (isset($this->queryParams["filter-first-letter-name"])) {
+           
+            if (strlen($queryFilter) === 0) {
+                $queryFilter = " WHERE a.name LIKE CONCAT(:firstLetterName, '%')";
+            }
+            else {
+                $queryFilter .= " AND a.name LIKE CONCAT(:firstLetterName, '%')";
+            }
+            
+            $params[':firstLetterName'] = $this->queryParams["filter-first-letter-name"];
+        }
+        if (isset($this->queryParams["filter-first-letter-state"])) {
+           
+            if(strlen($queryFilter) === 0) {
+                $queryFilter = " WHERE s.name LIKE CONCAT(:firstLetterState, '%')";
+            }
+            else{
```

Comment:
else should be on same line as }
example:
} else {

Reply:
fixed